payjoin/src/lib.rs

```diff
@@ -47,5 +47,5 @@ pub(crate) mod weight;
 
 #[cfg(feature = "base64")]
 pub use bitcoin::base64;
-pub use uri::{PjParseError, PjUri, PjUriBuilder, Uri};
+pub use uri::{MaybePayjoinExtras, PjParseError, PjUri, PjUriBuilder, Uri};
```

Comment:
this is useful in testing where we want to mutate the pjuri in order to change test direction(ie start with a pj uri and then maybe change it to something different) example: https://github.com/lightningdevkit/ldk-node/pull/257/files#diff-ebba69f49eedcd3dd6a656eff964c6184b6d29c02fbad6a49d3b41c87cc3de79R192